app/client/cli/helpers/common.go

```diff
@@ -0,0 +1,14 @@
+package helpers
+
+import (
+	"github.com/pokt-network/pocket/runtime"
+	"github.com/pokt-network/pocket/shared/modules"
+)
+
+var (
+	GenesisPath = runtime.GetEnv("GENESIS_PATH", "build/config/genesis.json")
+	RpcHost     string
```

Comment:
Can this be overriden by `RemoteCLIURL`? 

Asking to:
1. Understand
2. Modify / consolidate as needed